float32 dx_in_m
float32 lin_speed
---
bool success
